Closes #89: Setup Frontend README

AngelaLH opened the pull request:
Closes #89 

Frontend README set up containing:
- Prerequisites to install
- Getting started
- Available scripts
- Learn more

AngelaLH committed:
Merge pull request #1 from UOA-SE701-Group3-2021/main

Merge  with upstream

AngelaLH committed:
Closes #89: Setup Frontend README

RawiriHohepa commented on telos-frontend/README.md:
```diff
@@ -1,45 +1,56 @@
+# Getting Started
 
-### `npm start`
+1. Open the command prompt and set the working directory as the `3Lancers` directory of your cloned repository. (`cd 3Lancers`)
```
can you change the cd command to `cd path/to/3Lancers` just to emphasise that they are navigating to the right folder?

RawiriHohepa commented on telos-frontend/README.md:
```diff
@@ -1,45 +1,56 @@
 
-### `npm start`
+1. Open the command prompt and set the working directory as the `3Lancers` directory of your cloned repository. (`cd 3Lancers`)
+2. Run `npm install` to install all the required dependencies
```
Are we using `npm install` or `npm ci`? I'm pretty sure either one is fine as long as we keep it consistent among everyone - that may be what was causing the really big package-lock diffs

RawiriHohepa replied:
Based on offline discussion it seems that we'll stick with npm install

RawiriHohepa reviewed (changes requested):
Looks good! Just a small change and a decision for the team to make as a whole over whether to use `npm install` or `npm ci` - happy to approve once those are sorted!

ambiyu commented on telos-frontend/README.md:
```diff
@@ -1,45 +1,56 @@
+    - [npm](https://www.npmjs.com/get-npm)
+    - [Node.js](https://nodejs.org/en/)
+    - [Prettier](https://prettier.io/docs/en/install.html)
+    - [Lint](https://www.npmjs.com/package/lint)
```
Should be ESLint

AngelaLH committed:
Update README.md based on requested changes

*Lint changed to ESLint
*`cd 3Lancers` changed to `cd path/to/3Lancers`  in Getting Started to be more clear

RawiriHohepa reviewed (approved):
Looks good now, thanks Angela!

ambiyu reviewed (approved):
LGTM
